open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/src/test/java/org/odpi/openmetadata/userinterface/uichassis/springboot/service/OpenLineageServiceTest.java

```diff
@@ -120,6 +122,27 @@ public void testGlossaryLineage() throws PropertyServerException, InvalidParamet
         checkResponse(response);
     }
 
+    @Test
+    @DisplayName("Search")
+    public void search() throws PropertyServerException, InvalidParameterException, OpenLineageException {
+        LineageSearchRequest searchRequest = new LineageSearchRequest();
+        try {
+            List<LineageVertex> lineageVertices = new ArrayList<>(lineageVerticesAndEdges.getLineageVertices());
+            when(openLineageClient.search(USER_ID, searchRequest))
+                    .thenReturn(lineageVertices);
+        } catch (OpenLineageException e) {
+            e.printStackTrace();
```

Comment:
I would remove the catch clause (in case you don't need anything inside) and put a finally instead to do the needed checks